docs/document/content/user-manual/shardingsphere-proxy/startup/bin.en.md

```diff
@@ -45,6 +45,9 @@ please refer to [Distributed Transaction](/en/user-manual/shardingsphere-jdbc/sp
 When developer need to use user-defined algorithm, should use the way below to configure algorithm, use sharding algorithm as example. 
 
 1. Implement `ShardingAlgorithm` interface.
+1. Create META-INF/services directory in the resources directory.
+1. Create a new file org.apache.shardingsphere.sharding.spi.ShardingAlgorithm in the META-INF/services directory.
```

Comment:
resources and META-INF/services should around with , just keep consist with ShardingAlgorithm`